src/Aspire.ProjectTemplates/templates/aspire-empty/AspireApplication.1.ServiceDefaults/Extensions.cs

```diff
@@ -56,8 +55,9 @@ public static IHostApplicationBuilder ConfigureOpenTelemetry(this IHostApplicati
                 }
 
                 tracing.AddAspNetCoreInstrumentation()
-                       .AddGrpcClientInstrumentation()
-                       .AddHttpClientInstrumentation();
+                    // Uncomment the following line to enable gRPC instrumentation (requires the OpenTelemetry.Instrumentation.GrpcNetClient package)
```

Comment:
(super nit) the whitespace is now inconsistent in this file. Above on line 46-47, the `.` aligns with the `.` above it. Here the next line is starting 4 spaces in.

Reply:
lol yeah. It's a mix of the correct formatting and "Fowler-formatting". I'll fix it all to be correct 😄